src/js/components/Button/doc.js

```diff
@@ -92,7 +109,9 @@ with plain Buttons.`,
     href: PropTypes.string.description(
       'If specified, the button will behave like an anchor tag.',
     ),
-    icon: PropTypes.element.description('Icon element to place in the button.'),
+    icon: PropTypes.element.description(`Icon element to place in the button. 
+    For accessibility with screen readers, if using just an icon (no label), 
+    add an aria-label that describes the icon.`),
```

Comment:
Similar question/comment related to `aria-label` vs `a11yTitle` plus specificity as to where to add.

Reply:
Updated to a11yTitle since this is what we document.